### ISSUE TITLE
Panorama Snapshot

### ISSUE THREAD

**Issue by PeterGhimself** [space-concordia-robotics/robotics-prototype#113]
Create a panorama saved image to be able to view later.

- [ ] Determine best method to take pictures pre-panorama stitching
- - [ ] Create function to take images before stitching
- - [ ] Explore Logitech C920 H.264 pre-processing capability

- [x] Find method to stitch images together
- - [x] Create function to stitch images together using OpenCV
- - [x] Test function with multiple resolutions (480p/1080p)

- [ ] Implement function on the rover as an action
- [ ] Implement SCP to transfer final image from rover to basestation computer
- [ ] Integrate functionality into GUI, local test
- [ ] Integrate with the system (blocked by camera servo issues not being resolved yet)

**Comment by vashmata** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Panoramic Photography](https://trello.com/c/wNUiw9RZ/55-panoramic-photography)

**Comment by PeterGhimself** [comment]
@KoaTheKoala any plans on continuing work here or is this on hold?

**Comment by MartensCedric** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [A24.7 Panorama Snapshot](https://trello.com/c/Z9WBNe1S/163-a247-panorama-snapshot)
